scripts/quick-deploy-service.ts

```diff
@@ -206,7 +210,11 @@ function getDockerInfo(): { registry?: string; username?: string } {
 
 function buildDockerImageAndPush(
   username: string,
-  serviceId: 'media-service' | 'catalog-service' | 'entitlement-service',
+  serviceId:
```

Comment:
is vod-2-live not buildable via hosting v3?

Reply:
Good point. So far it was not but I will add it.